pages/c/_cluster/apps/install.vue

```diff
@@ -342,26 +191,39 @@ export default {
       nameDisabled:        false,
       openApi:             true,
       resetValues:         false,
-      defaultTab:          'appReadme',
       showPreview:         false,
+      preShowPreview:      false,
       showDiff:            false,
       showValuesComponent: true,
       showQuestions:       true,
+      showSlideIn:         false,
       componentHasTabs:    false,
       wait:                true,
 
       historyMax: 5,
       timeout:    600,
 
-      catalogOSAnnotation: CATALOG_ANNOTATIONS.SUPPORTED_OS,
+      steps: [{
+        name:        'basics',
+        label:       this.t('catalog.install.steps.basics.label'),
+        subtext:     this.t('catalog.install.steps.basics.subtext'),
+        ready:       true,
+      }, {
+        name:        'helmValues',
+        label:       this.t('catalog.install.steps.helmValues.label'),
+        subtext:     this.t('catalog.install.steps.helmValues.subtext'),
+        ready:       true,
+      }, {
+        name:        'helmCli',
+        label:       this.t('catalog.install.steps.helmCli.label'),
+        subtext:     this.t('catalog.install.steps.helmCli.subtext'),
+        ready:       true,
```

Comment:
It's pretty rare for there to be a reason for a normal user to change these Helm settings, so making it a third step I have to click through every time seems excessive...

Maybe they can be combined with step 1 as "Helm Options" (the name/namespace/etc kinda go with that) and leave only step 2: Chart Options?  They could probably even be hidden behind a Show/Hide Advanced Options toggle in there.

Reply:
In progress

Reply:
This is now an optional step triggered by a checkbox in the first step